clipper_admin/clipper_manager.py

```diff
@@ -1103,10 +1157,6 @@ def _put_container_on_host(self, container_name):
                 "docker images -q {cn}".format(cn=container_name))
 
             if len(local_result.stdout) > 0:
-                saved_fname = container_name.replace("/", "_")
-                subprocess.call("docker save -o /tmp/{fn}.tar {cn}".format(
-                    fn=saved_fname, cn=container_name))
-                tar_loc = "/tmp/{fn}.tar".format(fn=saved_fname)
                 self._execute_put(tar_loc, tar_loc)
```

Comment:
`tar_loc` is now undefined yet is being referenced. Why were these lines removed?